scripts/AbinsModules/Instruments/NoneInstrument.py

```diff
@@ -0,0 +1,149 @@
+import numpy as np
+import math
+
+from Instrument import Instrument
+from AbinsModules import AbinsParameters
+from AbinsModules.KpointsData import KpointsData
+
+class NoneInstrument(Instrument):
```

Comment:
Please select a different name for this child class, i.e. different from NoneInstrument